tests/_Func.js

```diff
@@ -0,0 +1,82 @@
+// import { assert } from 'chai';
```

Comment:
Remove this file.

Reply:
are you talking about the line?
import { assert } from 'chai'